shopyo/themes/front/ecommerceus/styles.css

```diff
@@ -0,0 +1,47 @@
+
+    html {
+     scroll-behavior: smooth;
+        }
+    .suggest {
+
+    }
+
```

Comment:
better to remove any empty selectors and also would make it more readable if you can correct the indentation in this style.css file.